docs/fundamentals/toc.yml

```diff
@@ -358,10 +358,565 @@ items:
   - name: Code analysis
     items:
     - name: Overview
-      href: productivity/code-analysis.md
+      href: code-analysis/overview.md
       displayName: code analysis, analyzers
     - name: Configure rules
-      href: productivity/configure-code-analysis-rules.md
+      href: code-analysis/configure-rules.md
+    - name: Code quality rule reference
+      items:
+      - name: Overview
+        href: code-analysis/rules/quality-rules-reference.md
+      - name: Design warnings
+        items:
+        - name: Overview
+          displayName: "design warnings"
```

Comment:
```suggestion
          displayName: "design rules"
```

Reply:
Probably good to just find and replace "warnings" with "rules" in the changed part of this doc...